packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -286,6 +297,80 @@ skipInNode(describe)('plugin-meetings', () => {
           throw e;
         }));
 
+      // FHD - remote video tests - starts
+      // Firefox bug - https://bugzilla.mozilla.org/show_bug.cgi?id=1537986
+      if (getBrowserName() !== 'Firefox') {
+        const getRemoteHeight = (minHeight = 360, maxHeight = 1080) => new Promise((resolve, reject) => {
+          testInterval = setInterval(() => {
+            const remoteSettings = bob.meeting.mediaProperties.remoteVideoTrack.getSettings();
+            const height = remoteSettings.height && +remoteSettings.height;
+
+            console.log(`Resolution for ${minHeight} ${maxHeight} : `, count, height);
+            if (height >= minHeight && height <= maxHeight) {
+              clearInterval(testInterval);
+              resolve(height);
+            }
+
+            if (count === INTERVAL_COUNT) {
+              clearInterval(testInterval);
+              if (!height) reject(new Error('Resolution undefined'));
+              else resolve(height);
+            }
+
+            count += 1;
+          }, WAIT_TIME);
+        });
+
+        it('alice resolution is 1080p, check what bob receives', async () => {
+          await testUtils.delayedPromise(alice.meeting.setLocalVideoQuality('1080p'));
+          const height = await getRemoteHeight(1080);
+
+          assert.isAbove(height, 500);
+          assert.isBelow(height, 1081);
+        });
+
+        it('bob requests 480p, should get resolution around 480p', async () => {
+          await testUtils.delayedPromise(bob.meeting.setRemoteQualityLevel('LOW'));
+          const height = await getRemoteHeight(400, 480);
+
+          assert.isAbove(height, 400);
+          assert.isBelow(height, 481);
+        });
+
+        it('bob upgrades remote to 720p, should get resolution around 720', async () => {
+          await testUtils.delayedPromise(bob.meeting.setRemoteQualityLevel('MEDIUM'));
+          const height = await getRemoteHeight(720, 1079);
```

Comment:
why the mismatch between the values here and in the asserts later?

Reply:
[Same reason](https://github.com/webex/webex-js-sdk/pull/2446#discussion_r986433243)